src/public/logger.h

```diff
@@ -22,7 +22,8 @@ class Logger {
 public:
   enum Level {
     ERROR = 0,
-    WARNING = 1
+    WARNING = 1,
+    INFO = 2
```

Comment:
At least in the applications I work with, i tend to use these log levels.

debug, detailed, normal, warning, fatal